src/chainparams.cpp

```diff
@@ -394,6 +394,15 @@ bool SelectParamsFromCommandLine()
     return true;
 }
 
+int64_t CChainParams::MaxTipAge() const {
+    if ( Params().NetworkIDString() == "regtest" )
+    {
+        int maxTipAge = (int)GetArg("-maxtipage", nMaxTipAge);
+        return maxTipAge;
```

Comment:
Any reason for the cast to `int` and then to `int64_t`? It looks like you can make it more compact as:

```suggestion
        return GetArg("-maxtipage", nMaxTipAge);
```
Also, please try to keep a style consistent with the rest of the file.

Reply:
Done